docs/7/7.4-ansible.md

```diff
@@ -1,3 +1,154 @@
 # Ansible
 
+Ansible has many use cases including provisioning, configuration management, application deployment, continuous delivery, security & compliance and orchestration.
 
+It uses easy to write configuration files to run these operations which lets enterprises manage infrastructure/configuration as code organization wide in version control.
```

Comment:
Replace this line with:
It uses easy to write configuration files to run these operations which lets enterprises manage infrastructure & configuration as code in version control.